.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -6,38 +6,45 @@ labels: enhancement
 assignees: ""
 ---
 
-### Description
+<!--
+
+⚠ Please note that this project is released with a Contributor Code of Conduct.
+By participating in this project you agree to abide by its terms.
+Read more: https://github.com/jilleJr/Newtonsoft.Json-for-Unity/blob/master/CODE_OF_CONDUCT.md
+
+All sections are just template. Remove if they don't help your case.
+
+-->
+
+## Description
 
 <!--
 Is your feature request related to a problem? Please describe.
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+Ex. I'm always frustrated when [...]
 -->
 
-### Why we need this
+## Motivation
 
 <!--
-Why should we put in time and effort for this?
-Do we gain time, style, or ease of use from this?
+Why do we need this?
+Ex: Would make builds faster.
 -->
 
-### Suggested solution
+## Suggested solution
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[heading-style] Headings should use setext](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=4743215)